shpc/main/modules/templates/docker.lua

```diff
@@ -34,6 +34,14 @@ For each of the above, you can export:
  - PODMAN_COMMAND_OPTS: to define custom options for the command
 ]]) 
 
+{% if not settings.default_version %}
```

Comment:
Any way to think about a common "include" or do you want me to look at this later?

Reply:
E.g., I wonder if we could do:

```lua
{% if not settings.default_version %}{% include 'default_version.lua' %}{% endif %}
```
and then dump the inner content with the check into it. I'm thinking if it's in the same directory it should find the file. We could also prepare for more duplication and make an includes directory:

```
singularity.lua
docker.lua
includes/
    default_version.lua
```
```lua
{% if not settings.default_version %}{% include 'includes/default_version.lua' %}{% endif %}
```

Reply:
That's a very good idea ! Would it work with the `{% if not settings.default_version %}` conditional in `default_version.lua` ?

Reply:
I'm actually testing now - I think the issue we will run into is that we are currently using Jinja2. Template, which takes a string directly. We do this because we can't always predict where the templates come from, and we need to run `self.substitute()` on the string content. So I think this would only work (using include) if we can find a way to tell Jinja "load the Template from this string, but also include these paths so the include statement will work". Traditionally I'd just use a FileSystemLoader and then have an environment load the template, but I've never done that for the jinja2.Template directly (trying to search for examples of doing this but no luck so far!)

Reply:
okay I got it working - it's kind of  terrible but I'll open a PR because minimally it would be fun to test :P